src/webview/common-ext/createComponentHelpers.ts

```diff
@@ -129,7 +129,8 @@ export function getDevfileRegistries(): DevfileRegistry[] {
     const components = ComponentTypesView.instance.getCompDescriptions();
     for (const component of components) {
         const devfileRegistry = devfileRegistries.find(
-            (devfileRegistry) => devfileRegistry.url === component.registry.url.toString(),
+             // eslint-disable-next-line @typescript-eslint/no-base-to-string
+             (devfileRegistry) => devfileRegistry.url === component.registry.url.toString(),
```

Comment:
you could swap `toString()` to `href` to avoid this linting error as well

Reply:
got it